xml/art-obs-beginners-guide.xml

```diff
@@ -1006,30 +1056,56 @@ Patch0:         &gitproject;_main.diff</screen>
     <para>Create the branch from &exampleuser_plain;'s home project:</para>
     <screen>&prompt.user2;<command>osc</command> bco &obshome1; &gitproject;</screen>
     <para>This will create a branched package in &obsa; at <uri>&obsbranch2;/&gitproject;</uri>
-     and will checkout a directory <filename>&obsworkdir2;/branches/home/&obsuser1;/&gitproject;</filename>.</para>
+     and will check out a directory
+     <filename>&obsworkdir2;/branches/home/&obsuser1;/&gitproject;</filename>.
+    </para>
    </step>
    <step>
     <para>Change the working directory to your checked out branch:</para>
     <screen>&prompt.user2;<command>cd</command> &obsworkdir2;/branches/home/&obsuser1;/&gitproject;</screen>
    </step>
    <step>
-    <para>Make your changes as shown in <xref
+    <para>Make changes as shown in <xref
      linkend="sec.obsbg.uc.patching"/>.
     </para>
    </step>
    <step>
-    <para>Build your package and check the result:</para>
+    <para>Build the package and check the result:</para>
+    <remark>
+     Why not just build for the default OS with all the default options, i.e.
+     just "osc build"? That would be way more generic. - sknorr, 2017-08-16
```

Comment:
Ok, I've added this distinction, but a bit before (search for the option `--local-package`).